src/vm/dllimportcallback.cpp

```diff
@@ -1416,6 +1416,7 @@ VOID UMThunkMarshInfo::RunTimeInit()
     {
         // caller pop
         m_cbRetPop = 0;
+        m_cbActualArgSize += numRegistersUsed * STACK_ELEM_SIZE;
```

Comment:
Hmm, that's really strange. The m_cbRetPop should be the same as the m_cbActualArgSize, since the native caller calling with stdcall convention expects the function to pop all of the arguments. And since the caller passes everything on stack, the two values should match. So it looks like there is some other problem that we've uncovered. Can you please tell me which tests were failing?